specification/cognitiveservices/data-plane/Form/stable/v1.0/Form.json

```diff
@@ -0,0 +1,731 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
+        "title": "Cognitive Service Form Recognizer API V1 (Preview)",
+        "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        }
+    },
+    "security": [
+        {
+            "apim_key": []
+        }
+    ],
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{Endpoint}/form/v1.0",
+        "useSchemePrefix": false,
+        "parameters": [
+            {
+                "$ref": "#/parameters/Endpoint"
+            }
+        ]
+    },
+    "paths": {
+        "/formrecognizer/v1.0/custom/train": {
+            "post": {
+                "tags": [
+                    "Form"
+                ],
+                "summary": "Train a model to analyze using a set of documents of supported types.",
+                "description": "The train request must include a 'Source' parameter that is either a Azure Storage Blob Container SAS Uri or a path to a locally mounted drive. When local paths are specified, they must always follow the Linux/Unix style absolute path convention and be rooted to the {Mounts:Input} configuration setting value.",
+                "operationId": "FormrecognizerV1.0CustomTrainPost",
```

Comment:
operationId should be "GroupName_OperationName"
Carefully choose GroupName and OperationName, they will greatly effect the auto-generated SDK/Doc.

Reply:
Formrecognizer_OpName